go.mod

```diff
@@ -11,6 +11,7 @@ require (
 	github.com/rancher/wrangler v0.7.3-0.20201020003736-e86bc912dfac
 	github.com/rancher/wrangler-api v0.6.1-0.20200427172631-a7c2f09b783e
 	github.com/sirupsen/logrus v1.4.2
+	github.com/stretchr/testify v1.4.0
```

Comment:
was there not a go.sum change here ?

Reply:
Looks like it, just tried `go mod tidy` tidy on main branch and nothing.